pandas/core/internals/blocks.py

```diff
@@ -1700,49 +1686,9 @@ def putmask(self, mask, new, align=True, inplace=False, axis=0,
         new_values = self._try_coerce_result(new_values)
         return [self.make_block(values=new_values)]
 
-    def _slice(self, slicer):
-        """ return a slice of my values (but densify first) """
-        return self.get_values()[slicer]
-
     def _try_cast_result(self, result, dtype=None):
         return result
 
-    def _unstack(self, unstacker_func, new_columns, n_rows, fill_value):
```

Comment:
this and the two methods above are in NonConsolodateableMixin, which is now only mixed in to ExtensionBlock, which directly defines these three methods.